firefox_ios/views/counters.view.lkml

```diff
@@ -1,22 +1,3 @@
-explore: app_display_major_version {
-  hidden: yes
-}
-
-view: app_display_major_version {
-  derived_table: {
-    sql:
-      SELECT DISTINCT cast(split(app_display_version, ".")[offset(0)] as int64) as app_display_major_version
-      FROM `moz-fx-data-shared-prod`.org_mozilla_ios_firefox.metrics m
-      WHERE DATE(submission_timestamp) > DATE_SUB(current_date(), interval 14 day)
-      ORDER BY 1 ;;
-  }
-
-  dimension: app_display_major_version {
-    type: number
-    sql:  ${TABLE}.app_display_major_version ;;
-  }
-}
-
```

Comment:
@fbertsch When I tried the counter explore with this explore as a suggestion for the new derived field, it didn't seem to come up with any results. Any ideas on what I did wrong here? I'd like to use a similiar approach for the auto-generated labeled counters in the future...